src/Validator/ProgrammeDateTimeNotInPastValidator.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Validator;
+
+use App\Entity\Programme;
+use Symfony\Component\HttpFoundation\File\Exception\UnexpectedTypeException;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class ProgrammeDateTimeNotInPastValidator extends ConstraintValidator
+{
+    public function validate($value, Constraint $constraint)
+    {
```

Comment:
also throw exception if `$value` not a date time

Reply:
Yes.